### PR TITLE
20 ivp d1

### LINKED ISSUE #20: Create schema for Initial Visit (IVP) form D1: Clinician Diagnosis
https://www.alz.washington.edu/NONMEMBER/UDS/DOCS/VER3/IVPforms/d1_ivp.pdf

### PR DESCRIPTION
**Pull request by SenseCode**
close #20 
image
image
image
image

### CODE REVIEW

File: src/encoded/schemas/ivp_d1.json [hunk 1]
```diff
@@ -0,0 +1,2148 @@
...
+                            
+                        },
+                        "amylpet": {
+                            "title": "6a. Abnormally elevated amyloid on PET",
+                            "type": "string",
+                            "enum": [
+                                "No 0",
+                                "Yes 1",
+                                "Unknown, not assessed 8"
+                            ]
```
Comment (mingjiecn) [inline src/encoded/schemas/ivp_d1.json:337]: for radio button, check the design book for numbering.
image

**Review by mingjiecn — COMMENTED** [review]
There are also some radio button in this form, need to check design book on redcap for numbering

File: src/encoded/schemas/ivp_d1.json (hunk 1 above)
Comment (SenseCode) [reply src/encoded/schemas/ivp_d1.json:337]: This one I talked to Venkat today, for mapping, if you prefer to put the number first, it's better we change the redcap version with put number first. So the option showing in page should be 
0 No, 1 Yes, 8 Unknown, not assessed
won't be:  0, No 0, 1, Yes 1, 8 Unknow, not assessed 8...

image
image
image
image

**Comment by SenseCode** [comment]
@mingjiecn I don't quite understand what you meant...  Normally in design book, the number before comma is the mapping number will be used in access data, it's for you to use. After the comma part is  for me to use in schema enum field. Maybe we can talk about it in channel or in meeting to the avoid the misunderstanding here.  
Not all the options have the pattern with number+ text,  we have to switch it or reedit the redcap form to help with mapping issues if you want number first as always. 
image

**Review by mingjiecn — APPROVED** [review]
looks great!

**Pull request merged**